tests/tests_integration/test_file_format_integration/test_xml_integration.py

```diff
@@ -150,7 +151,7 @@ def test_xml_reader_with_attributes(
 ):
     """Reading XML files with attributes works as expected"""
     spark_version = get_spark_version(spark)
-    if spark_version < (3, 0):
+    if spark_version < Version("3.0"):
```

Comment:
nit: можно чуть упростить
```suggestion
    if spark_version.major < 3:
```